.github/workflows/ci.yml

```diff
@@ -192,6 +192,15 @@ jobs:
           flags: "dev-unittests, ${{ matrix.test-tags }}, ${{ matrix.os }}"
           verbose: true
           use_oidc: true
+      - name: "Check schemas are up to date"
+        run: |
+          nix develop -L --no-update-lock-file --command just gen-schemas
+          if ! git diff --quiet cli/schemas; then
```

Comment:
Double checked this would fail by pushing a bad update to one of the schemas https://github.com/flox/flox/actions/runs/18567758581/job/52933438576#step:9:116